src/l500/l500-options.cpp

```diff
@@ -81,10 +81,10 @@ namespace librealsense
         if (_fw_version < firmware_version(MIN_CONTROLS_FW_VERSION))
         {
             depth_sensor.register_option
-            (RS2_OPTION_VISUAL_PRESET, std::make_shared<uvc_xu_option<int >>(raw_depth_sensor, ivcam2::depth_xu, ivcam2::L500_AMBIENT,
-                "Change the depth ambient light to ambient: 1 for no ambient and 2 for low ambient",
-                std::map<float, std::string>{ { float(RS2_AMBIENT_LIGHT_NO_AMBIENT), "No Ambient"},
-                { float(RS2_AMBIENT_LIGHT_LOW_AMBIENT), "Low Ambient" }}));
+            (RS2_OPTION_VISUAL_PRESET, std::make_shared<uvc_xu_option<int >>(raw_depth_sensor, ivcam2::depth_xu, ivcam2::L500_DIGITAL_GAIN,
+                "Change the depth digital gain to: 1 for high gain and 2 for low gain",
+                std::map<float, std::string>{ { float(RS2_DIGITAL_GAIN_HIGH_GAIN), "High Gain"},
```

Comment:
Interesting... it doesn't take the string representation from the get_string() functions?

Reply:
It was implemented this way with ambient light (I don't know why) so I kept it.
Should I change?